scss/mixins/_debugcolors.scss

```diff
@@ -0,0 +1,16 @@
+$map: (
```

Comment:
This filename consists of two words, but are joined together. "Grid settings" are two words as well, but that file is named "grid-settings". Rename this file to `_debug-settings.scss` for consistency.

Reply:
Your right! thanks